nixos/modules/system/boot/loader/limine/limine-install.py

```diff
@@ -362,9 +362,15 @@ def main():
                 efibootmgr = os.path.join(config('efiBootMgrPath'), 'bin', 'efibootmgr')
                 efi_partition = find_mounted_device(config('efiMountPoint'))
                 efi_disk = find_disk_device(efi_partition)
+
+                efibootmgr_output = subprocess.check_output([efibootmgr], stderr=subprocess.STDOUT, universal_newlines=True)
+                create_flag = '-c'
+                if matches := re.findall(r'Boot[0-9a-fA-F]{4} Limine', efibootmgr_output):
+                    create_flag = '-C' # Don't change boot order
```

Comment:
```suggestion
                efibootmgr_output = subprocess.check_output([efibootmgr], stderr=subprocess.STDOUT, universal_newlines=True)
                create_flag = '-c'
                # Check the output of `efibootmgr` to find if limine is already installed and present in the boot record
                if matches := re.findall(r'Boot[0-9a-fA-F]{4} Limine', efibootmgr_output):
                    create_flag = '-C' # if present, keep the same boot order
```

perhaps explain a bit :) just for future maintenance